zoombot/zoombot/api.go

```diff
@@ -49,6 +51,36 @@ type GlobalDialInNumbers struct {
 	Type        string `json:"type"`
 }
 
+// Get User
+
+type GetUserResponse struct {
```

Comment:
why dont' we just have the fields we need so we never parse any user data we don't need

Reply:
I set this up so we could easily use these functions for whatever, but that's a good point. I'll get rid of the unnecessary stuff. Do you think I should comment it out or just axe it entirely?

Reply:
Never mind, I'm just going to axe the extra stuff and add a comment.

Reply:
perfect!